src/common/components/header/index.js

```diff
@@ -7,7 +7,7 @@ import styles from './header.css';
 
 const wordmark = 'https://assets.ubuntu.com/v1/d45097a4-snapcraft.io-logotype.svg';
 
-class Header extends Component {
+export class Header extends Component {
```

Comment:
We should be binding the store further up the tree and passing in props using JSX instead of exporting twice. Subtle differences like this often lead to difficult-to-debug bugs - especially when some components export to default and some to named exports.